src/VisualStudio/Core/Impl/CodeModel/FileCodeModel.cs

```diff
@@ -37,6 +38,8 @@ public sealed partial class FileCodeModel : AbstractCodeModelObject, EnvDTE.File
             bool isSourceGeneratorOutput,
             ITextManagerAdapter textManagerAdapter)
         {
+            // Keep track that code model was accessed.  We want to get a sense of how widespread usage of it still is.
+            Logger.Log(FunctionId.CodeModel_FileCodeModel_Create);
```

Comment:
This should be fine, but note that the file code model objects are long-lived. If your goal is to just to figure out what percentage of sessions use CodeModel, this is fine. If you want to figure out something a bit more precise (like how often is CodeModel used within a sesson), you'll need to put this somewhere else.

Reply:
just trying to figure out the former.  really, what percentage of users hit this.